### ISSUE TITLE
Migrate to Debian 11

### ISSUE THREAD

**Issue by MatmaRex** [MatmaRex/patchdemo#516]
Previous migration: #233

Needed to properly resolve #500 and #501.

We should get a bigger disk to avoid running out of space again like #505.

**Comment by MatmaRex** [comment]
* Created a new instance `patchdemo3`
* Added web proxy https://patchdemo3.wmcloud.org/
* Added volume `patchdemo3-wikis` following https://wikitech.wikimedia.org/wiki/Help:Adding_Disk_Space_to_Cloud_VPS_instances#Cinder:_Attachable_Block_Storage_for_Cloud_VPS – mounted on `/srv`
* Installed Patch Demo

I didn't copy the existing wikis or configuration yet.

**Comment by MatmaRex** [comment]
Copied everything over, mostly following the notes at #233.

Copied the files using `matmarex@patchdemo2:/var/www/html/wikis$ tar cf - . | pv | ssh matmarex@patchdemo3 "tar x -C /var/www/html/wikis/"` (13.3GiB 0:24:02).

https://patchdemo.wmflabs.org/ now points at the new instance.

I will keep the old instance around for some weeks in case I did something wrong and we need to bring it back.

**Comment by MatmaRex** [comment]
Closing notes:
* The `/` disk (holding everything including the repositories and databases) is not actually bigger, it's about 20 GB just like the old one. There doesn't seem to be an easy way to set up an instance differently. Currently it's 56% used (compared to 86% on the old server) – I don't really know what's causing the difference. If it ends up growing too much, we should figure out how to move either repositories (I tried and the setup script didn't work) or databases (I didn't try).
* The `/src` disk (holding the wikis) is actually smaller, 80 GB now compared to 140 GB before. 80 GB is the default per-project limit. We only have 14 GB of wikis, so this is fine. (And they supposedly can be resized with minimal effort.)
* The instance itself is less powerful than the previous one, 4 CPU 8 GB RAM compared to 8 CPU 16 GB, because we hit the limits on these resources as well. Despite that it works just fine and in fact creating the wikis seems significantly faster (around 1 minute with the default settings, compared to about 3.5 minutes before). I think the old one was overpowered because that was the only way to get a bigger disk, but currently one can attach more disk space to any instance.
